libs/application/templates/accident-notification/src/hooks/useLazyGetIdentity.ts

```diff
@@ -0,0 +1,22 @@
+import { gql } from '@apollo/client'
```

Comment:
No need for this, we need to use the apollo client usequery for this https://github.com/island-is/island.is/blob/main/libs/application/templates/accident-notification/src/fields/ApplicationStatus/index.tsx#L45